packages/nanocurrency/src/keys.ts

```diff
@@ -87,6 +87,12 @@ export function derivePublicKey(secretKeyOrAddress: string): string {
 export interface DeriveAddressParams {
   /** Whether to use nano_ instead of xrb_ */
   useNanoPrefix?: boolean
+  /** Whether to use NOS prefix nos_ instead of xrb_ or nano_ */
+  nos?: boolean
```

Comment:
I'd be more comfortable with a `useFork?: Fork` param, with `Fork` enum.

Reply:
Yeah that is prob much easier to support them without having to worry about testing etc. 

Thanks